### PR TITLE
Add Rails 7.1 in test suite + compatibility

### PR DESCRIPTION
**Pull request by ericproulx**
Add Rails 7.1 in test suite + compatibility

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 #### Features
 
+* [#2353](https://github.com/ruby-grape/grape/pull/2353): Add rails 7.1 in test suite + compatibility - [@ericproulx](https://github.com/ericproulx).
```
Comment (dblock) [inline CHANGELOG.md:5]: Nit: let's say "Added Rails 7.1 support"?

**Review by dblock — APPROVED** [review]
Looks great! Nit on CHANGELOG. Do we need to make any updates to README? We say a few things about beta versions of Rails, maybe we should add something about Rails 7?

**Comment by ericproulx** [comment]
> Looks great! Nit on CHANGELOG. Do we need to make any updates to README? We say a few things about beta versions of Rails, maybe we should add something about Rails 7?

Maybe something about the railtie for Rails 7. I'll write something

**Comment by ericproulx** [comment]
> > Looks great! Nit on CHANGELOG. Do we need to make any updates to README? We say a few things about beta versions of Rails, maybe we should add something about Rails 7?
> 
> Maybe something about the railtie for Rails 7. I'll write something

@dblock [done](https://github.com/ruby-grape/grape/pull/2353/commits/9976fd85d2f45712b49a19ddc1bf3b75537aec9d)

**Comment by dblock** [comment]
Merged, thanks for taking care of it.

**Pull request merged**